frontend/src/views/admin/SubscriberPanelView.vue

```diff
@@ -273,4 +319,15 @@ onMounted(async () => {
   <div v-else class="flex size-full min-h-[75vh] items-center justify-center">
     <loading-spinner/>
   </div>
+  <!-- Refresh link confirmation modal -->
+  <confirmation-modal
+    :open="hardDeleteModalOpen"
+    title="WARNING"
+    message="This will completely remove a user from Thunderbird Appointment, including all of their user data. Don't do this unless you're told to."
+    confirm-label="Delete"
```

Comment:
Localize those would be nice too!

Reply:
Ah yeah, they're internal only so wasn't a priority. There's a few things in the admin panel aren't localized. So didn't want to bother you with that part 😅 

I'll create a follow up ticket.

Reply:
Perfect, thanks!